CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.7 (not released yet)
 
+- fix revealing secret with default defined, where environment variable is defined, it would always default.
```

Comment:
This could properly be better worded if anyone has any suggestion :raising_hand_man:

Reply:
:wave: @darxriggs perhaps you have some :sweat_smile: